### PR TITLE
Update approver and maintainer practices

### PR DESCRIPTION
**Pull request by svrnm**

### CODE REVIEW

File: content/en/docs/contributing/development.md [hunk 1]
```diff
@@ -131,9 +131,27 @@ approvers and maintainers while doing code reviews:
...
+  - PRs that auto-update versions in the registry can be fixed, approved and
+    merged immediately
+  - PRs that auto-update the versions of SDKs, zero-code instrumentations or the
+    collector can be approved and merged except the corresponding SIG signals
+    that merging should be postponed
+  - PRs that auto-update the version of any specification often require updates
+    to scripts for the CI checks to pass. In that case often
+    [@chalin](https://github.com/chalin/) will handle that PR. Otherwise those
+    PRs can as well be approved and merged except the corresponding SIG signals
+    that merging should be postponed
```
Comment (theletterf) [inline content/en/docs/contributing/development.md:151]:
```suggestion
    that merging should be postponed.
```

**Review by theletterf — APPROVED** [review]

File: content/en/docs/contributing/development.md [hunk 2]
```diff
@@ -148,11 +166,31 @@ approvers and maintainers while doing code reviews:
...
+  [OTel Code of Conduct](https://github.com/open-telemetry/community/?tab=coc-ov-file#opentelemetry-community-code-of-conduct)
+  and are friendly and helpful towards contributors. In the case of a conflict,
+  misunderstanding or any other kind of situation that makes an
+  approver/maintainer feel uncomfortable they can step back from a conversation,
+  issue or PR and ask another approver/maintainer to step in.
+
+The following workflow can be applied by maintainers to merge PRs:
+
+- Make sure that a PR has all approvals and all CI checks pass
+- If the branch is out-of-date, update it via the GitHub UI.
```
Comment (chalin) [inline content/en/docs/contributing/development.md:187]: I'd suggest a rebase update. Merge updates can cause problems for long-running PRs.

File: content/en/docs/contributing/development.md [hunk 3]
```diff
@@ -131,9 +131,27 @@ approvers and maintainers while doing code reviews:
...
+    This signals to the SIG that they need to handle the PR
   - If no SIG approval is given within a certain grace period (two weeks in
     general, but may be less in urgent cases), docs maintainer may use their own
     judgement to merge that PR
+- PRs created by bots can be merged by the following practice:
+  - PRs that auto-update versions in the registry can be fixed, approved and
+    merged immediately
+  - PRs that auto-update the versions of SDKs, zero-code instrumentations or the
+    collector can be approved and merged except the corresponding SIG signals
+    that merging should be postponed
```
[the remark this answers is not in the document]
Comment (chalin) [reply content/en/docs/contributing/development.md:146]: IIRC it happening once or twice in the past couple of years. I can't recall for sure how we (docs folks) were notified, but it was either through Slack or through an issue/PR or issue/PR comment either in this repo or in the SIG repo (with a notification of @open-telemetry/docs-approvers). 

I'm not sure that we need to say how a SIG signals though. Your call.

[the remark this answers is not in the document]
Comment (svrnm) [reply content/en/docs/contributing/development.md:146]: I left it vague on purpose, I would expect them to write an issue comment or ping us via slack, but other means of communication are also acceptable.

File: content/en/docs/contributing/development.md (hunk 2 above)
Comment (svrnm) [reply content/en/docs/contributing/development.md:187]:
```suggestion
- If the branch is out-of-date, rebase update it via the GitHub UI.
```

**Review by tiffany76 — APPROVED** [review]
Makes sense to me. Thanks!

**Comment by theletterf** [comment]
> Warning: Unknown word (chalin)

😅 I'll fix that and merge.

**Pull request merged**